sql/src/test/scala/org/apache/sedona/sql/functionTestScala.scala

```diff
@@ -335,6 +335,18 @@ class functionTestScala extends TestBaseScala with Matchers with GeometrySample
 
     }
 
+    it("Passed ST_XMax") {
+      var test = sparkSession.sql("SELECT ST_XMax(ST_GeomFromWKT('POLYGON ((-3 -3, 3 -3, 3 3, -3 3, -3 -3))'))")
```

Comment:
Please use a more significant test case here. E.g., POLYGON ((-1 -11, 0 10, 1 11, 2 12, -1 -11))

Reply:
Changed the test case

Reply:
Changed the test case.